Automattic/wp-calypso
# Limited Global Styles: Open launchbar popup by default

Fixes https://github.com/Automattic/dotcom-forge/issues/2268

## Proposed Changes

Improves the Limited Global Styles notice displayed in the launchbar:
- The dialog is open by default when custom styles are previewed, so it's less likely to miss.
- Wording has been adapted to make the UX more intuitive.

State | Before | After
--- | --- | ---
Preview ON | <img width="959" alt="Screenshot 2023-05-01 at 15 20 00" src="https://user-images.githubusercontent.com/1233880/235458472-67044f6c-237a-432a-88a1-b6bc0f85922e.png"> | <img width="994" alt="Screenshot 2023-05-01 at 15 17 55" src="https://user-images.githubusercontent.com/1233880/235458306-ff676b8d-e3be-4e3c-b23b-e30824d20681.png">
Preview ON (mobile) | <img width="409" alt="Screenshot 2023-05-01 at 15 19 40" src="https://user-images.githubusercontent.com/1233880/235458664-e98e322f-55dd-48e0-83e8-fbeed5ef8aaf.png"> | <img width="401" alt="Screenshot 2023-05-01 at 15 18 44" src="https://user-images.githubusercontent.com/1233880/235458904-87a2c093-16e5-4bab-bad1-4c1c736c7de0.png">
Preview OFF | <img width="962" alt="Screenshot 2023-05-01 at 15 20 27" src="https://user-images.githubusercontent.com/1233880/235458692-4cb17d1c-fb94-4cff-a0ab-0e30229aa5e6.png"> | <img width="996" alt="Screenshot 2023-05-01 at 15 18 14" src="https://user-images.githubusercontent.com/1233880/235458736-499bce2d-8e3f-41cc-8b1a-5342c3a89bc3.png">
Preview OFF (mobile) | <img width="416" alt="Screenshot 2023-05-01 at 15 19 32" src="https://user-images.githubusercontent.com/1233880/235458775-69c35f2a-42b3-4208-ad33-cb09f6b94d6f.png"> | <img width="421" alt="Screenshot 2023-05-01 at 15 18 32" src="https://user-images.githubusercontent.com/1233880/235458937-420cff34-5cc4-4552-8227-5347ac172ec7.png">

## Testing Instructions

- Apply these changes to your sandbox.
- Sandbox a free site that uses Global Styles.
- Visit the frontend.
- Make sure the dialog is open by default.
- Make sure you can show the custom styles using the secondary button in the dialog.
- Make sure the wording is clear.
